httpcore/_async/connection_pool.py

```diff
@@ -105,7 +105,7 @@ def __init__(
 
         self._ssl_context = SSLContext() if ssl_context is None else ssl_context
         self._max_connections = max_connections
-        self._max_keepalive = max_keepalive
```

Comment:
We were setting our state from the deprecated setting.

httpx is passing `max_keepalive_connections`, which won't be having any effect at the moment.